src/universal/containers/EditingStatus/EditingStatusContainer.js

```diff
@@ -24,7 +25,7 @@ const makeEditingStatus = (editors, active, updatedAt) => {
     makeEditingStatus.active = active;
     // no one else is editing
     if (editors.length === 0) {
-      makeEditingStatus.cache = active ? <span>editing<Ellipsis/></span> : fromNow(updatedAt);
+      makeEditingStatus.cache = active ? <span>editing<Ellipsis/></span> : fromNowString(updatedAt);
```

Comment:
You're going to have to merge from `master` and resolve a few conflicts in this file before we can merge this in. I don't think it should take too long...